src/lib/zexe_backend/zexe_backend_common/endoscale_round_5_wires.ml

```diff
@@ -5,15 +5,15 @@ module H_list = Snarky_backendless.H_list
 module Stable = struct
   module V1 = struct
     type 'a t =
-      { b2: 'a
-      ; xt: 'a
-      ; b1: 'a
+      { mutable b2: 'a
```

Comment:
Can you rework the code so that these aren't mutable? I seem to remember doing the same for the last set of changes. You can use `{acc with b2= ...; xt= ...; ...}` to do a non-mutable update, or it may make sense to use mutability per variable and fill in the record at the end.